src/development/tools/devtools/console.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Using the Debug console
+description: Learn how to use the DevTools console to xxx.
+---
+
+The DevTools Debug console allows you to watch an
+application's standard output (`stdout`),
+evaluate expressions for a paused or running
+app in debug mode, and analyze inbound and outbound
+references for objects.
+
+{{site.alert.note}}
+  This page is up to date for DevTools 2.23.0.
+{{site.alert.end}}
+
+The Debug console is available from the [Inspector][],
+[Debugger][], and [Memory][] views. To bring up
+the debug console, click the **More info** link,
+as shown:
+
+![Screenshot of how to bring up Debug console]({{site.url}}/assets/images/docs/tools/devtools/debug-console.png){:width="100%"}
+
+[Inspector]: {{site.url}}/development/tools/devtools/inspector
+[Debugger]:  {{site.url}}/development/tools/devtools/debugger
+[Memory]:    {{site.url}}/development/tools/devtools/memory
+
+## Watch application output
+
+The console shows the application's standard output (`stdout`):
+
+![Screenshot of stdout in Console view]({{site.url}}/assets/images/docs/tools/devtools/console-stdout.png)
+
+## Explore inspected widgets
+
+If you click a widget on the **Inspector** screen,
+the variable for this widget displays in the **Console**: 
+
+![Screenshot of inspected widget in Console view]({{site.url}}/assets/images/docs/tools/devtools/console-inspect-widget.png){:width="100%"}
+
+## Evaluate expressions
+
+In the console, you can evaluate expressions for a paused
+or running application, assuming that you are running
+your app in debug mode:
+
+![Screenshot showing evaluating an expression in the console]({{site.url}}/assets/images/docs/tools/devtools/console-evaluate-expressions.png)
+
+To assign an evaluated object to a variable,
+use $0, $1 … $5:
```

Comment:
Let's make it easier for people who cannot recognize text on images:

use $0, $1 … $5 in the form `var x = $0`:

Reply:
Are they limited to 5?